package.json

```diff
@@ -27,9 +27,9 @@
     "react-dom": "^0.14.7"
   },
   "devDependencies": {
+    "babel-core": "^6.4.5",
```

Comment:
Must have missed saving it. Must not have set npm to auto-save deps on this machine, since it is installed locally.